facebookads/test/unit.py

```diff
@@ -112,6 +112,36 @@ def test_builds_from_object(self):
         obj = ei.build_objects_from_response(response)
         assert len(obj) == 1 and obj[0]['id'] == "601957/targetingsentencelines"
 
+    def test_builds_from_object_with_data_key(self):
+        """
+        Sometimes the response returns a single JSON object — with a "data".
```

Comment:
Sorry, our linters are pretty strict. Could you modify the hyphen here to be a normal `-` ?